packages/stencil/src/components/datacom-overflow-menu/datacom-overflow-menu.tsx

```diff
@@ -0,0 +1,139 @@
+import {
+  Component,
+  Host,
+  h,
+  Prop,
+  State,
+  Element,
+  Listen,
+} from '@stencil/core';
+import { OverflowMenuIcon } from './assets/overflow-menu-icon';
+
+export type MenuSizeType = 'standard' | 'small';
+export type OverFlowMenuVariantType = 'horizontal' | 'vertical';
+export type MenuItemsPositionType = 'center' | 'left' | 'right';
+
+@Component({
+  tag: 'datacom-overflow-menu',
+  styleUrl: 'datacom-overflow-menu.css',
+  shadow: true,
+})
+export class DatacomOverflowMenu {
+  @Prop() variant: OverFlowMenuVariantType = 'horizontal';
+  @Prop() label: string;
+  @State() isOpen: boolean = false;
+  @Prop() size: MenuSizeType = 'small';
+  @Element() hostElement: HTMLDatacomOverflowMenuElement;
+  @Prop({ mutable: true }) position: MenuItemsPositionType = 'center';
+  private buttonRef: HTMLButtonElement;
+  private firstElementRef: HTMLElement;
+
+  private open() {
+    this.isOpen = true;
+  }
+  private close() {
+    this.isOpen = false;
+  }
+
+  handleKeyUp = (event: KeyboardEvent) => {
```

Comment:
Please make it to private function.

Reply:
In StencilJS, it's not necessary to explicitly declare component methods as private because all component methods are private by default. StencilJS encapsulates component logic within the component class, making its methods and properties accessible only within the component itself.  https://stenciljs.com/docs/methods